src/lando/api/tests/test_landings.py

```diff
@@ -777,29 +829,32 @@ def test_format_single_success_changed(
     ), "Autoformat via amending should only land a single commit."
 
 
+def _scm_get_previous_hash(scm: AbstractSCM) -> str:
+    if scm.scm_type() == SCM_TYPE_HG:
+        return HgSCM.run_hg(scm, ["log", "-r", "tip^", "-T", "{node}"]).decode("utf-8")
+    return GitSCM._git_run("rev-parse", "HEAD^", cwd=scm.path)
+
+
+@pytest.mark.parametrize(
+    "repo_type",
+    [
+        SCM_TYPE_GIT,
+        SCM_TYPE_HG,
+    ],
+)
 @pytest.mark.django_db
 def test_format_stack_success_changed(
-    hg_server,
-    hg_clone,
+    repo_mc,
     treestatusdouble,
     monkeypatch,
     create_patch_revision,
+    repo_type: str,
 ):
     """Test formatting a stack via an autoformat tip commit."""
-    tree = "mozilla-central"
-    treestatusdouble.open_tree(tree)
-    repo = Repo.objects.create(
-        scm_type=SCM_TYPE_HG,
-        name=tree,
-        url=hg_server,
-        push_path=hg_server,
-        pull_path=hg_server,
-        required_permission=SCM_LEVEL_3,
-        autoformat_enabled=True,
-        system_path=hg_clone.strpath,
-    )
-
-    hgrepo = HgSCM(hg_clone.strpath)
+    repo = repo_mc(repo_type)
+    repo.autoformat_enabled = (True,)
```

Comment:
Same issue here regarding adding these as overrides to a fixture, and how they are not saved.